api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -258,11 +259,7 @@ private User initializeUserIfNeeded() {
 
   @Override
   public ResponseEntity<Profile> createAccount(CreateAccountRequest request) {
-    if (request.getInvitationKey() == null
```

Comment:
This key needs to be checked during account creation. Otherwise, I can create an account without an invitation key by using curl.